sheepdog/utils/transforms/graph_to_doc.py

```diff
@@ -842,4 +842,9 @@ def format_prop(prop):
                 row.append(node[prop] or '')
             # Tack on the linked properties.
             row.extend(map(lambda col: col or '', result[1:]))
+            # Convert row elements to string if they are not
+            for idx, val in enumerate(row):
+                if ~isinstance(val, str):
```

Comment:
might be clearer to do: `if not isinstance(val, str):`

Reply:
fixed in 57428c7cc5836594a79e5f7e3a46ea15381587a0